### PR TITLE
Fixes #54 - Upgrade to Kinto server v1.3.

### LINKED ISSUE #54: Demo is broken.
Related https://github.com/mozilla-services/kinto/issues/126

### PR DESCRIPTION
**Pull request by n1k0**
- [x] Update demo code to use Kinto dev server instance running 1.3
- [x] Fix issue #56
- [x] Update dist files

### CODE REVIEW

**Comment by n1k0** [comment]
r=? @Natim @leplatrem

**Comment by Natim** [comment]
r+

File: src/api.js [hunk 1]
```diff
@@ -158,10 +158,7 @@ export default class Api {
         results.conflicts.push({
           type: "outgoing",
           local: records[index],
-          // TODO: Once we get record information in this response object,
-          // add it; for now, that's the error json body only.
-          // Ref https://github.com/mozilla-services/kinto/issues/122
-          remote: response.body
+          remote: response.body.details && response.body.details.existing || null
```
Comment (leplatrem) [inline src/api.js]: nit³: ``|| null`` not tested :)

**Comment by leplatrem** [comment]
r+

**Pull request merged**